src/components/Cards/Recipe-Card/Recipe-Card.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+
+export default function RecipeCard(props) {
```

Comment:
it is better to make props like this 
```suggestion
export default function RecipeCard({title, image, id, calories, ...etc}) {
```